portugues/revisa/tenta-nao-consegue/tenta-falha.md

```diff
@@ -1,5 +1,7 @@
-Voc� envia uma resposta ao desenvolvedor, dizendo que n�o conseguiu reproduzir o bug.
+Voc� envia uma resposta ao desenvolvedor, dizendo que n�o conseguiu reproduzir o bug.
 
 O desenvolvedor:
 
 [Te responde e adota um comportamento estranho!](eh-perseguido/salve-se-quem-puder.md)
+[Te responde de forma amigável!](amigavel/amigavel.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.